pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.linkedin.sparktfrecord</groupId>
-    <artifactId>spark-tfrecord_2.11</artifactId>
+    <artifactId>spark-tfrecord</artifactId>
```

Comment:
Current convention is to append scala version. such as
https://search.maven.org/search?q=a:spark-core_2.11
https://search.maven.org/search?q=a:spark-core_2.12
Otherwise, users don't know which scala version it was compiled with.

Reply:
Agree, we'll also publish jars to maven repo soon, package name with Scala version suffix would make it explicit to users.